assets/javascripts/lib/discourse-markdown/discourse-chat-transcript.js

```diff
@@ -15,6 +15,7 @@ const chatTranscriptRule = {
     let [username, messageIdStart, messageTimeStart] =
       (tagInfo.attrs.quote && tagInfo.attrs.quote.split(";")) || [];
     let multiQuote = !!tagInfo.attrs.multiQuote;
+    let noLink = !!tagInfo.attrs.noLink;
```

Comment:
Actually I have no idea why everything's `let` in this chunk given they're not mutated. 🤔